src/windows/MediaProxy.js

```diff
@@ -33,8 +42,11 @@ module.exports = {
     create:function(win, lose, args) {
         var id = args[0];
         var src = args[1];
+        var createAudioNode = args[2];
         var thisM = Media.get(id);
 
+        src = setTemporaryFsByDefault(src);
```

Comment:
Addressed in https://github.com/MSOpenTech/cordova-plugin-media/commit/41d6149b7b55a6a8ce067d0a1a30ace3ca636231